app/src/main/java/crazydude/com/telemetry/ui/MapsActivity.kt

```diff
@@ -1188,4 +1193,19 @@ class MapsActivity : AppCompatActivity(), DataDecoder.Listener {
             switchToConnectedState()
         }
     }
+
+    private val mBatInfoReceiver: BroadcastReceiver = object : BroadcastReceiver() {
```

Comment:
can you please rename this to batInfoReceiver. The google naming guidelines is not needed in current year

Reply:
Done